docs/src/rules/no-invalid-this.md

```diff
@@ -284,6 +284,51 @@ obj.Foo = function Foo() {
 
 :::
 
+This rule additionally supports TypeScript type syntax.
+
+Examples of **incorrect** TypeScript code for this rule:
+
+:::incorrect
+
+```ts
+/*eslint no-invalid-this: "error"*/
```

Comment:
Let's also add examples with `@this` tag

Reply:
Done!